docker/Dockerfile

```
@@ -26,33 +23,40 @@ RUN echo "---- INSTALL RUNTIME PACKAGES ----" && \
 
 # Add ffmpeg, tone, mp4v2 and fdkaac
 COPY --from=ffmpeg /usr/local/bin/ffmpeg /usr/local/bin/
 COPY --from=ffmpeg /usr/local/bin/ffprobe /usr/local/bin/
-# COPY --from=tone /usr/local/bin/tone /usr/local/bin/
 COPY --from=mp4v2 /usr/local/bin/mp4* /usr/local/bin/
 COPY --from=mp4v2 /usr/local/lib/libmp4v2* /usr/local/lib/
 COPY --from=fdkaac /usr/local/bin/fdkaac /usr/local/bin/
 
 ARG M4B_TOOL_DOWNLOAD_LINK="https://github.com/sandreas/m4b-tool/releases/latest/download/m4b-tool.tar.gz"
 
 RUN echo "---- INSTALL M4B-TOOL ----" \
```

Hadolint (2.14.0)

[info] Avoid use of wget without progress bar. Use `wget --progress=dot:giga <url>`. Or consider using `-q` or `-nv` (shorthands for `--quiet` or `--no-verbose`).

(DL3047)